resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/submit/KubernetesClientApplication.scala

```diff
@@ -136,9 +136,17 @@ private[spark] class Client(
     val driverPodName = resolvedDriverPod.getMetadata.getName
 
     var watch: Watch = null
-    val createdDriverPod = kubernetesClient.pods().create(resolvedDriverPod)
+    var createdDriverPod: Pod = null
     try {
-      val otherKubernetesResources = resolvedDriverSpec.driverKubernetesResources ++ Seq(configMap)
+      createdDriverPod = kubernetesClient.pods().create(resolvedDriverPod)
+    } catch {
+      case NonFatal(e) =>
+        logError("Fail to create driver pod, you may use wrong master URL.")
```

Comment:
BTW, the error message looks misleading. The following can fail in many reasons. And, many user failures are due to the permissions of namespace instead of a wrong master URL.
```
kubernetesClient.pods().create(resolvedDriverPod)
```

Reply:
yeah, make sense.
How about changing to this? Any suggestion?
```scala
logError("Fail to create driver pod, you may use wrong master URL or permission denied. 
Please check the network connection and permission")
```